src/w3c/headers.js

```diff
@@ -697,6 +698,36 @@ export function run(conf) {
   });
 }
 
+async function deriveHistoryURI(conf) {
+  if (!conf.shortName || conf.historyURI === null) {
+    return; // Nothing to do
+  }
+
+  const historyURL = new URL(
+    conf.historyURI ?? conf.shortName,
+    "https://www.w3.org/standards/history/"
+  );
+
+  // If it's on the Rec Track or it's TR worthy, then it has a history.
+  const willHaveHistory = [...recTrackStatus, ...W3CNotes, ...maybeRecTrack];
+  if (willHaveHistory.includes(conf.specStatus)) {
+    conf.historyURI = historyURL.href;
+    return;
+  }
+
+  // Do a fetch HEAD request to see if the history exists...
+  // We don't discriminate... if it's on the W3C website with a history,
+  // we show it.
+  try {
+    const response = await fetch(historyURL, { method: "HEAD" });
+    if (response.ok && response.status === 200) {
```

Comment:
`.ok` should be enough? Why test 200 specifically (and not 304 etc.)?

Reply:
yeah, agree... I was thinking of other 20X cases, but 🤷‍♂️